packages/kbn-cli-dev-mode/package.json

```diff
@@ -15,7 +15,6 @@
   },
   "dependencies": {
     "@kbn/config": "link:../kbn-config",
-    "@kbn/config-schema": "link:../kbn-config-schema",
```

Comment:
NIT: Please feel free to dismiss it if it makes no sense: Instead of removing, should we keep it as `link:../../bazel-bin/packages/kbn-config-schema`? This way we don't lose the reference to it as a dependency (making it easier to create the BUILD.bazel file in the future).
What do you think?

Reply:
Those declared dependencies are only needed in order to kbn-pm legacy build system be able to build the correct order when running `bootstrap`. While we could keep it to make it easier to create the BUILD.bazel file for `kbn-cli-dev-mode` in the future, the Bazel compilation will still error at the time which will make us able to know what are the missing dependencies 😃

Reply:
@afharo is that okay?

Reply:
Fair enough. I see you've got a plan already. Feel free to dismiss my comment :)

Also, thanks for explaining!

Reply:
@afharo can you approve the PR then? 😄